pkgs/tools/X11/jumpapp/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, perl, pandoc, fetchFromGitHub, xdotool, wmctrl, xprop, nettools }:
+
+stdenv.mkDerivation rec {
+  name = "jumpapp-${version}";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "mkropat";
+    repo = "jumpapp";
+    rev = "v${version}";
+    sha256 = "11ibh51q4vcjkz9fqyw5dy9qrkqxm42hpdccas1s6h2dk9z62kfb";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
+  buildInputs = [ perl pandoc ];
+  postFixup = let
+    runtimePath = lib.makeSearchPath "bin" [ xdotool wmctrl xprop nettools perl ];
+  in
+  ''
+    sed -i "2 i export PATH=${runtimePath}:\$PATH" $out/bin/jumpapp
+    sed -i "2 i export PATH=${perl}/bin:\$PATH" $out/bin/jumpappify-desktop-entry
+  '';
+
+  meta = {
+    homepage = https://github.com/mkropat/jumpapp;
+    description = "A run-or-raise application switcher for any X11 desktop";
+    license = stdenv.lib.licenses.mit;
+    maintainers = [ ];
```

Comment:
would you kind sir add yourself as a maintainer, please?

Reply:
done

Reply:
thanks!